### PR TITLE
1.2.10 release notes

### PR DESCRIPTION
**Pull request by deathbeam**
Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

### CODE REVIEW

File: src/_posts/2017-12-21-1.2.10-Release.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+title: '1.2.10 Release'
+description: 'New 1.2.10 release of RuneLite is out. We added many new features, like native UI, notifications and few plugin improvements.'
+---
+
+In the new release we added a lot of new functionality to the RuneLite to make
```
Comment (UniquePassive) [inline src/_posts/2017-12-21-1.2.10-Release.md]: Remove "the"

File: src/_posts/2017-12-21-1.2.10-Release.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+the end-user experience even better, and to compete with concurent third-party
```
Comment (UniquePassive) [inline src/_posts/2017-12-21-1.2.10-Release.md]: Replace "concurent" with "well established"?

File: src/_posts/2017-12-21-1.2.10-Release.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+clients.
+
+The most notable new feature is the **new UI design**. I tried to make it look as
+close to the native OSRS interfaces as possible. Here is the end result:
+
+![native ui](/img/blog/native-ui.png)
+
+The UI was updated for all of the current RuneLite plugins, and the new API for
+it is really easy to use, so it can be used in future plugins too. One example
+of such plugin using the new UI api is the new **Team Capes** plugin.
```
Comment (UniquePassive) [inline src/_posts/2017-12-21-1.2.10-Release.md]: Change to "An example of a plugin using the new"?

File: src/_posts/2017-12-21-1.2.10-Release.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+from monster, or just simply to check price of stack of your precious magic
+logs in bank. That is why there were quite a few improvements to examine plugin
+in the client. Now you are able to examine items and item stacks in you bank,
+inventory, equipment screen or even in the Miscellania screen and it shows
+Grand Exchange and High Alchemy price of that item in the game chat. Here is
+small example:
+
+![examine plugin](/img/blog/examine-plugin.png)
+
+Also, to make talking to your clan mates easier, there is new clan chat plugin
```
Comment (devinfrench) [inline src/_posts/2017-12-21-1.2.10-Release.md]: Also was already used as a transition word. I suggest using something else or remove transition words altogether.

File: src/_posts/2017-12-21-1.2.10-Release.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+The most notable new feature is the **new UI design**. I tried to make it look as
+close to the native OSRS interfaces as possible. Here is the end result:
+
+![native ui](/img/blog/native-ui.png)
+
+The UI was updated for all of the current RuneLite plugins, and the new API for
+it is really easy to use, so it can be used in future plugins too. One example
+of such plugin using the new UI api is the new **Team Capes** plugin.
+
+The plugin is showing count of players wearing specific team capes currently
```
Comment (UniquePassive) [inline src/_posts/2017-12-21-1.2.10-Release.md]: count -> the amount

File: src/_posts/2017-12-21-1.2.10-Release.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+visible on the map, very useful for pking clans. Here is how it looks in game:
```
Comment (UniquePassive) [inline src/_posts/2017-12-21-1.2.10-Release.md]: Add "which is" before "very useful for pking clans" 🙂

File: src/_posts/2017-12-21-1.2.10-Release.md (hunk 1 above)
Comment (UniquePassive) [reply src/_posts/2017-12-21-1.2.10-Release.md]: or add "client" after RuneLite

**Pull request merged**